aries-site/src/data/structures/components.js

```diff
@@ -845,7 +845,7 @@ export const components = [
   },
   {
     name: 'Menu',
-    accessibility: 'Failed WCAG 2.2 AA',
+    accessibility: 'Passed WCAG 2.2 AA',
```

Comment:
what about the menu rule in the components.json file did we change that to passing?

Reply:
Yeah the menu rule is updated already, I think Taylor may have updated it when she did changed the site to use the new theme

Reply:
gotcha yes just verified it is